README.md

```diff
@@ -22,43 +26,44 @@ Azure IoT Edge extension <sup>Preview</sup> makes it easy to code, build, deploy
+- [Python for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+
 We will soon support other languages and more services on IoT Edge.
 
 ## Commnads
 
 Press `F1` or `Ctrl + Shift + P` to open command palette, type `Edge:` to see all the commands:
 - **Edge: Setup Edge**: Setup the Edge runtime.
 - **Edge: Start Edge**: Start Edge runtime in your local machine.
-- **Edge: Build IoT Edge module**: Build Edge module from source code. 
-- **Edge: Build IoT Edge module Docker image**: Containerize your Edge module to Docker image. 
-- **Edge: Push IoT Edge module Docker image**: Push an image to a Docker registry.
+- **Edge: New IoT Edge Solution**: Create an IoT Edge solution.
+- **Edge: Add IoT Edge Module**: Add a new IoT Edge module to the IoT Edge solution.
+- **Edge: Build IoT Edge Module Image**: Containerize Edge module from source code. 
+- **Edge: Build and Push IoT Edge Module Image**: Containerize and push Edge module image to a Docker registry.
+- **Edge: Build IoT Edge Solution**: Build and push all the Edge module image in the solution and expand deployment manifest. 
+- **Edge: Generate IoT Edge Deployment Manifest**: Generate the deployment manifest file for Edge deployment from deployment template.
 - **Edge: Create deployment for Edge device**: Create and submit the deployment to your Edge device with specified deployment manifest. 
 - **Edge: Setup Edge using configuration file**: Take a configuration file for Edge setup.
 - **Edge: Generate Edge setup configuration file**: Generate the json file for Edge: setup Edge using configuration file. Open the json file to see further description.
-- **Edge: Generate Edge deployment manifest**: Generate the json file for Edge deployment.
 - **Edge: Stop Edge**: Stop Edge runtime.
 - **Edge: Restart Edge**: Restart the Edge runtime.
 - **Edge: Uninstall**: Remove all modules and generated files.
 - **Edge: Log in to container registry**: Add registry credentials to Edge runtime.
 
 You can also trigger following frequently-used commands in context menu.
-- **Edge: Build IoT Edge module**: The context menu of the \*.csproj file in VS Code file explorer. Directly build the target \*.csproj IoT Edge C# module project.
-- **Edge: Build IoT Edge module Docker image**: The context menu of the Dockerfile or Dockerfile.debug. Use target Dockerfile to build a Docker image.
+- **Edge: Add IoT Edge Module**: The context menu of deployment.template.json file in VS Code file explorer. A new module will be added to the **modules** folder.
+- **Edge: Build IoT Edge Solution**: The context menu of deployment.template.json file in VS Code file explorer.
+- **Edge: Generate Edge setup configuration file**: The context menu of deployment.template.json file in VS Code file explorer. The deployment manifest (deployment.json) will be expanded from deployment.template.json.
```

Comment:
Should be "The context menu of a device in the Azure IoT Hub Devices explorer. A setup configuration file will be generated with the selected device's connection string."